R/tab-book.R

```diff
@@ -48,7 +51,7 @@
 #' @importFrom jsonlite fromJSON
 #' @export
 tabBook <- function(multitable, dataset, weight = crunch::weight(dataset),
-                    output_format = c("json", "xlsx"), file, filter = NULL,
+                    output_format = c("json", "xlsx"), file = NULL, filter = NULL,
```

Comment:
There was some code that I'm unfamiliar with the functionality of used previously for the `file` parameter. I flipped this to NULL instead of empty (with a subsequent test for `missing()`). The effects of this are unclear to me but the code appeared equivalent.

Reply:
Yeah, it's hard to pass missing arguments like the old API to other functions (like `tabBookMulti()`/`tabBookSingle()` so I agreed with your decision to change to `NULL`.